### ISSUE TITLE
Crash when entering no DoH URL

### ISSUE THREAD

**Issue by csagan5** [bromite/bromite#430]
### Bromite version

Version: `78.0.3904.119`
Arch: `x86`
Android version: `9.0`
Device model: `Nexus 5X`

### Is this bug about the SystemWebView?
No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?

No.

### Is the bug a crash?

Yes.

### Describe the bug

Crash when saving an empty URL for DoH.

### Steps to reproduce the bug

1. Go to the Secure DNS (DoH) configuration page
2. Edit the URL and delete everything from the text box
3. Press save
4. Any navigation request will crash the browser with:

```
12-05 20:41:36.098  5610  5610 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
12-05 20:41:36.098  5610  5610 F DEBUG   : Build fingerprint: 'google/sdk_gphone_x86/generic_x86:10/QPP6.190730.005.B1/5775370:userdebug/dev-keys'
12-05 20:41:36.098  5610  5610 F DEBUG   : Revision: '0'
12-05 20:41:36.098  5610  5610 F DEBUG   : ABI: 'x86'
12-05 20:41:36.098  5610  5610 F DEBUG   : Timestamp: 2019-12-05 20:41:36+0100
12-05 20:41:36.098  5610  5610 F DEBUG   : pid: 5376, tid: 5437, name: NetworkService  >>> org.bromite.bromite <<<
12-05 20:41:36.098  5610  5610 F DEBUG   : uid: 10137
12-05 20:41:36.098  5610  5610 F DEBUG   : signal 8 (SIGFPE), code 1 (FPE_INTDIV), fault addr 0xb5c08d57
12-05 20:41:36.098  5610  5610 F DEBUG   :     eax 00000000  ebx b9c686cc  ecx d5f92e80  edx 00000000
12-05 20:41:36.098  5610  5610 F DEBUG   :     edi c01c8180  esi 00000002
12-05 20:41:36.098  5610  5610 F DEBUG   :     ebp 00000000  esp bdcc06e0  eip b5c08d57
12-05 20:41:36.098  5610  5610 F DEBUG   : 
12-05 20:41:36.098  5610  5610 F DEBUG   : backtrace:
12-05 20:41:36.098  5610  5610 F DEBUG   :       #00 pc 00888d57  /data/app/org.bromite.bromite-1G0WIf1oVRc_8HSWcIaOBA==/base.apk!libchrome.so (offset 0xc7c000) (BuildId: 0119dba0b99e819a14a59e9571338dec88991283)
```

**Comment by AnkS4** [comment]
Crash after switching on DoH:

https://del.dog/mopesinazu.txt

**Comment by csagan5** [comment]
Fixed in `78.0.3904.130`
